homeassistant/components/motion_blinds/__init__.py

```diff
@@ -146,7 +146,10 @@ def stop_motion_multicast(event):
                 _LOGGER.debug("Shutting down Motion Listener")
                 multicast.Stop_listen()
 
-            hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, stop_motion_multicast)
+            unsub = hass.bus.async_listen_once(
+                EVENT_HOMEASSISTANT_STOP, stop_motion_multicast
+            )
+            entry.async_on_unload(unsub)
```

Comment:
We only want to unsubscribe this if we're unloading the last loaded config entry.

Reply:
You are totally right, I changed it now.